x/emissions/keeper/keeper.go

```diff
@@ -1916,7 +1916,8 @@ func (k *Keeper) InsertNetworkLossBundleAtBlock(
 	if err := types.ValidateBlockHeight(block); err != nil {
 		return errorsmod.Wrap(err, "block height validation failed")
 	}
-	if err := lossBundle.Validate(); err != nil {
+	err := lossBundle.Validate()
+	if err != nil {
```

Comment:
```suggestion
	if err := lossBundle.Validate(); err != nil {
```
Maybe better to keep it as it was